cpp/box/Box.h

```diff
@@ -93,12 +99,6 @@ class Box
         return !(*this == b);
     }
 
-    //! Set L, box lengths, inverses.  Box is also centered at zero.
-    void setL(const vec3<float>& L)
-    {
-        setL(L.x, L.y, L.z);
-    }
-
```

Comment:
Why is this no longer needed?

Reply:
It's not exported to python, nor was it used anywhere in the C++ code. The name clashes with a different method named `setL`, which nanobind cannot resolve.